maventa/maventa.test.ts

```diff
@@ -0,0 +1,27 @@
+// import { sendInvoiceToMyCompany } from './maventa';
+import { listInvoices } from './maventa';
+
+describe('Maventa Integration Tests', () => {
```

Comment:
It would be best to match the name of the file, so it's easier to detect which file the test came from.

Reply:
...or name of the class